hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -607,6 +607,20 @@ public final class OzoneConfigKeys {
       OZONE_OM_SNAPSHOT_COMPACTION_DAG_PRUNE_DAEMON_RUN_INTERVAL =
       "ozone.om.snapshot.compaction.dag.prune.daemon.run.interval";
 
+  public static final String
```

Comment:
Nit: these config keys and values are related to OM and should be added to [OMConfigKeys](https://github.com/apache/ozone/blob/184bfc21189583f857280933270f134dcac68879/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java#L30).

Previously OM related configs added here should be moved to OMConfigKeys as well but not as part of this PR. Can be done as part of [HDDS-8322](https://issues.apache.org/jira/browse/HDDS-8322).

Reply:
done